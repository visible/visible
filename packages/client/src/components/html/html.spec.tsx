import React from 'react';
import { render } from '@testing-library/react';
import { Html } from '.';

describe('Html', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <Html
        state={{ foo: 'bar' }}
        content="hello"
        manifest={{ 'main.js': '/dist/foobar.js' }}
        elements={[<meta key="my meta" content="test"></meta>]}
      />,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
