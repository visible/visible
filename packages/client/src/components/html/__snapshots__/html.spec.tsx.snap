// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Html matches snapshot 1`] = `
<html>
  <head>
    <meta
      charset="UTF-8"
    />
    <meta
      content="width=device-width, initial-scale=1.0"
      name="viewport"
    />
    <meta
      content="ie=edge"
      http-equiv="X-UA-Compatible"
    />
    <meta
      content="test"
    />
  </head>
  <body>
    <div
      id="root"
      role="application"
    >
      hello
    </div>
    <script>
      window.__APOLLO_STATE__={"foo":"bar"};
    </script>
    <script
      src="/dist/foobar.js"
      type="text/javascript"
    />
  </body>
</html>
`;
